templates/shared/issuelist.tmpl

```diff
@@ -4,7 +4,7 @@
 		<div class="flex-item flex-item-baseline">
 			<div class="flex-item-leading gt-ac">
 				{{if $.CanWriteIssuesOrPulls}}
-					<input type="checkbox" autocomplete="off" class="issue-checkbox gt-mr-4" data-issue-id={{.ID}} aria-label="{{$.locale.Tr "repo.issues.action_check"}} &quot;{{.Title}}&quot;">
+					<input dir="auto" type="checkbox" autocomplete="off" class="issue-checkbox gt-mr-4" data-issue-id={{.ID}} aria-label="{{$.locale.Tr "repo.issues.action_check"}} &quot;{{.Title}}&quot;">
```

Comment:
Doesn't really make sense on checkbox, but doesn't hurt either, so I think we keep it.